defold-spine/commonsrc/vertices.cpp

```diff
@@ -35,6 +37,89 @@ static uint32_t EnsureArrayFitsNumber(dmArray<T>& array, uint32_t num_to_add)
     return prev_size;
 }
 
+
+void GetSkeletonBounds(const spSkeleton* skeleton, SpineModelBounds& bounds)
+{
+    dmArray<float> scratch; // scratch buffer
+
+    // a "negative" bounding rectangle for starters
+    bounds.minX = FLT_MAX;
+    bounds.minY = FLT_MAX;
+    bounds.maxX = FLT_MIN;
+    bounds.maxY = FLT_MIN;
```

Comment:
This is a common misconception.
FLT_MIN is the minimum _positive_ representable floating point number.

Instead, use `-FLT_MAX`

This is also generally, why I don't use "float.h" for this, but just a large enough number, e.g. `1000000.0f`, then it becomes obvious.